plotly/offline/offline.py

```diff
@@ -90,6 +75,11 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly'):
                                 of the chart that will export the chart to
                                 Plotly Cloud or Plotly Enterprise
     link_text (default='Export to plot.ly') -- the text of export link
+    validate (default=True) -- validate that all of the keys in the figure
+                               are valid? omit if your version of plotly.js
+                               has become outdated with your version of
+                               graph_reference.json or if you need to include
+                               extra, unnecessary keys in your figure.
```

Comment:
sold. okie dokie. so then `/offline` will now have:

* `offline/plotly.min.js`
* `offline/plot-schema.json`

And somehow we'll need to route validation through the appropriate `plot-schema.json`?